### PR TITLE
visual changes

### PR DESCRIPTION
**Pull request by fieldsco**

### CODE REVIEW

File: src/components/Breadcrumb/style.scss [hunk 1]
```diff
@@ -45,31 +45,36 @@
   max-width: 100%;
 
   .icon-arrow-68 {
-    margin: 0 6px;
+    margin: 0 3px;
```
Comment (fieldsco) [inline src/components/Breadcrumb/style.scss]: I chose this value by doing an inspection on the actual dimension of the visible part of the svg (11px), so this/2 = 5 rounded down. we need 8, so I added an additional 3px.

File: src/components/Breadcrumb/style.scss [hunk 2]
```diff
@@ -45,31 +45,36 @@
...
+      color: $primary-4;
       font-size: 14px;
       font-style: normal;
       position: relative;
     }
 
-    &:last-child {
-      padding-right: 8px;
-
-      @media #{$media-small-minus} {
```
Comment (fieldsco) [inline src/components/Breadcrumb/style.scss]: not sure why these padding rules were here, when you decreased the screen it looked weird (see live reactions for what I'm talking about)

**Comment by fieldsco** [comment]
[(1) Adjust visual display of breadcrumb items](https://trello.com/c/MITaBeFz/1263-1-adjust-visual-display-of-breadcrumb-items)

File: src/components/Breadcrumb/style.scss (hunk 1 above)
Comment (samageloff) [reply src/components/Breadcrumb/style.scss]: 👍

File: src/components/Breadcrumb/style.scss (hunk 2 above)
Comment (samageloff) [reply src/components/Breadcrumb/style.scss]: That's fine. 👍

Comment (samageloff) [reply src/components/Breadcrumb/style.scss]: this is needed so that when the user is scrolling (horizontally), the last breadcrumb is visible.

**Pull request merged**